icc/icc_defs.mk

```
@@ -17,7 +17,7 @@ DEFAULT_TEST_CMD       = ./icctest$(EXESUFX)
 DEFAULT_OPENSSL_TEST_CMD = cd $(OSSL_DIR); make tests; apps/openssl speed
 # This is actually used to build an rc file on Windows
 DEFAULT_ASMOBJS        =
-DEFAULT_EXTRAS         =
+DEFAULT_EXTRAS         = PKCS11 PKCS11_PERF
```

```suggestion
DEFAULT_EXTRAS         =
```
PKCS11 is not a part of OCKC so remove those references - i.e. dont change them.